### PR TITLE
Add Cron Trigger template to Examples and clarify docs

### PR DESCRIPTION
**Pull request by Albertozhao**
This includes a note on setting up cron syntax in Wrangler.

### CODE REVIEW

File: products/workers/src/content/examples/cron-trigger.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    // Fetch some data
+    // Update API
+  return new Response("OK")
+}
+    /* 
+    If you're doing this in Wrangler, set the cron syntax (once per hour shown below) by adding this to your wrangler.toml:
+    
+    [triggers]
+    crons = ["0 * * * *"] 
+    */
```
Comment (lukeed) [inline products/workers/src/content/examples/cron-trigger.md]: Some funky spacing going on here

Comment (Albertozhao) [reply products/workers/src/content/examples/cron-trigger.md]: De-funked. Thank you

**Review by Albertozhao — COMMENTED** [review]
@lukeed de-funked the spacing, thank you

**Comment by lukeed** [comment]
Slam dunked the funk 😆 

Can we move the `If you're doing this in Wrangler, set the cron syntax` part into an actual paragraph and then  put the `[triggers]` part in a TOML code block?

I'm not running this locally so I'm not sure if it'll break the formatting and/or if it needs to be in the `ContentColumn` tags 🤷

**Review by kristianfreeman — APPROVED** [review]

File: products/workers/src/content/examples/cron-trigger.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+# Cron Triggers
+
+<ContentColumn>
+  <p>{props.frontmatter.summary}</p>
+</ContentColumn>
+
+```js
+addEventListener('scheduled', event => {
+    event.waitUntil(triggerEvent(event.request))
```
Comment (ObsidianMinor) [inline products/workers/src/content/examples/cron-trigger.md]: Scheduled events don't have requests or accept responses, so the example should probably do something else (likely with `event.scheduledTime` or `event.cron`)

**Comment by WalshyDev** [comment]
Could it also be clarified for how to do this in different environments? Something like:
If you wanted to deploy to a development environment you would do
[env.dev.triggers]
crons = ["0 * * * *"]

File: products/workers/src/content/examples/cron-trigger.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+async function triggerEvent(request) {
+    // Fetch some data
+    // Update API
+    return new Response("OK")
+}
+```
+
+## Setting Cron Triggers in Wrangler
+If you're deploying with Wrangler, you can set the cron syntax (once per hour shown below) by adding this to your `wrangler.toml`:
```
Comment (deadlypants1973) [inline products/workers/src/content/examples/cron-trigger.md]:
```suggestion
If you are deploying with Wrangler, set the cron syntax (once per hour as shown below) by adding this to your `wrangler.toml` file:
```

File: products/workers/src/content/platform/cron-triggers/index.md [hunk 4]
```diff
@@ -66,6 +66,12 @@ Here are some common time intervals that may be useful for setting up your Cron
 
 </Definitions>
 
+<Aside header="Testing Cron Triggers and potential delays">
```
Comment (deadlypants1973) [inline products/workers/src/content/platform/cron-triggers/index.md]:
```suggestion
<Aside type="note" header="Testing Cron Triggers and potential delays">
```

File: products/workers/src/content/platform/cron-triggers/index.md [hunk 5]
```diff
@@ -66,6 +66,12 @@ Here are some common time intervals that may be useful for setting up your Cron
...
+
+A recommended way for testing your Cron Trigger is to first deploy it to a test domain. Adding a new Cron Trigger, updating an old Cron Trigger, or deleting a Cron Trigger may take minutes for changes to propagate to the edge.
+
+</Aside>
+
 ## Viewing past events
 
 Users can view the execution history of their Cron Triggers on the Past Events section of the [Workers Triggers tab](https://dash.cloudflare.com/?to=/:account/workers) or through Cloudflare's [GraphQL Analytics API](https://developers.cloudflare.com/analytics/graphql-api).
```
Comment (deadlypants1973) [inline products/workers/src/content/platform/cron-triggers/index.md]:
```suggestion
Users can review the execution history of their Cron Triggers on the Past Events section of the [Workers Triggers tab](https://dash.cloudflare.com/?to=/:account/workers) or through Cloudflare's [GraphQL Analytics API](https://developers.cloudflare.com/analytics/graphql-api).
```

**Review by deadlypants1973 — APPROVED** [review]
Approved with comments for review.

**Pull request merged**